cucumber-expressions/java/pom.xml

```diff
@@ -17,29 +17,32 @@
 
     <properties>
         <project.Automatic-Module-Name>io.cucumber.cucumberexpressions</project.Automatic-Module-Name>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
I don't think its needed but it doesn't do any harm

Reply:
Not needed because it's defined `cucumber-parent` right?